tests/python/pytorch/nn/test_nn.py

```diff
@@ -1066,6 +1066,22 @@ def test_gated_graph_conv_one_etype(g, idtype):
     assert h.shape[-1] == 10
 
 
+@parametrize_idtype
+@pytest.mark.parametrize("g", get_cases(["homo"], exclude=["zero-degree"]))
+def test_gatedgcn_conv(g, idtype):
+    ctx = F.ctx()
+    g = g.astype(idtype).to(ctx)
+    gatedgcnconv = nn.GatedGCNConv(10, 10, 5)
+    feat = F.randn((g.num_nodes(), 10))
+    efeat = F.randn((g.num_edges(), 10))
+    gatedgcnconv = gatedgcnconv.to(ctx)
+
+    h, edge_h = gatedgcnconv(g, feat, efeat)
+    # current we only do shape check
+    assert h.shape == (g.number_of_dst_nodes(), 5)
+    assert edge_h.shape == (g.number_of_edges(), 5)
```

Comment:
Could you move this function to a new file `tests/python/pytorch/nn/conv/test_gatedgcnconv.py`?